website/docs/patterns/button-alignment/partials/specifications/order.md

```diff
@@ -0,0 +1,77 @@
+## Order
+
+When performing an action or function, users generally think from highest to lowest priority and from positive to negative. How buttons and actions are ordered should respect this by ordering the primary (most important) action first, secondary action second, and if applicable, tertiary action last.
+
+!!! Do
+
+Order buttons based on priority starting with the **highest** priority action (usually by using a primary button) to **lowest** priority (either secondary or tertiary actions).
+!!!
+
+![Ordering from high priority](/assets/patterns/button-alignment/order-high-low-priority.png)
+
+### Destructive actions
+
+In the case of a destructive action, the primary button should be replaced with a destructive or critical action button in the same position. While a destructive action may have a **negative** connotation, the priority of the action takes precedence over whether the action is positive or negative in this scenario.
+
+!!! Do
+
+![Destructive action order](/assets/patterns/button-alignment/order-destructive-action.png)
```

Comment:
in the example just above, the image is out of the `Do/Dont` box, here inside. maybe use a similar approach for both?